ReleaseNotes-2.8

```diff
@@ -30,6 +30,7 @@ Features
  * New obsservicedispatch service to handle source service runs in a queue
    and asynchron.
  * Allow admins to lock or delete users and their home projects via new command
+ * Users can be declared alias of other users. Useful for automated scripts.
```

Comment:
Better if we don't use "alias", just say something like "User can have subaccounts, state of subaccounts is dependent on the owner user. Useful for automated scripts"